modules/UI/UI.js

```diff
@@ -227,8 +231,8 @@ UI.initConference = function () {
 
     APP.store.dispatch(checkAutoEnableDesktopSharing());
 
-    if(!interfaceConfig.filmStripOnly) {
-        Feedback.init(eventEmitter);
+    if (interfaceConfig.filmStripOnly) {
+        APP.store.dispatch(shouldShowPostCallFeedbackDialog(false));
```

Comment:
Sorry for the confusion. I was just trying to get rid of this. So in the context of feedback on hangup, shouldn't we just check interfaceConfig.filmStripOnly when someone calls our openFeedbackDialog? Now that I'm thinking some more, eventually I would like us to end up with some Filmstrip-only mode initiation in react, which will probably do the same thing as above, set some properties to some components, instead of every component trying to "know" about filmstrip-only, so not sure anymore. You may ignore :)